src/npfctl/npfctl.8

```diff
@@ -136,6 +136,29 @@ If no mask is specified, a single host is assumed.
 List all entries in the currently loaded table specified by
 .Ar tid .
 This operation is expensive and should be used with caution.
+
+.It Ic table Ar tid Ic replace Oo Fl n Ar newid Oc Oo Fl t Ar type Oc Aq Ar path
```

Comment:
When the manual page is changed/updated, the practice is to bump its date, see the `.Dd` line in the top of this file.